modules/ROOT/pages/index.adoc

```diff
@@ -46,31 +44,31 @@ For Anypoint Platform PCE Version 3.0, the following policies are included with
 | Detokenization | 1.1.1
 | Header Injection | 1.2.1
 | Header Removal | 1.0.3
-| HTTP Caching | 1.0.3
-| IP Blacklist | 1.2.2
-| IP Whitelist | 1.2.2
+| HTTP Caching | 2.0.2
+| IP Blocklist | 1.0.0
+| IP Allowlist | 1.0.0
 | JSON Threat Protection | 1.1.4
 | JWT Validation | 1.1.3
 | Message Logging | 1.0.0
-| Mule OAuth 2.0 Access Token | 1.3.2
-| OpenAM OAuth 2.0 Token Enforcement | 1.3.2
-| OpenID Connect OAuth 2.0 Token Enforcement | 1.3.2
-| PingFederate Access Token Enforcement | 1.3.2
-| Rate Limiting | 1.3.4
-| Rate limiting - SLA based | 1.2.6
+| Mule OAuth 2.0 Access Token | 1.5.0
+| OpenAM OAuth 2.0 Token Enforcement | 1.5.0
+| OpenID Connect OAuth 2.0 Token Enforcement | 1.5.0
+| PingFederate Access Token Enforcement | 1.5.0
+| Rate Limiting | 1.3.5
+| Rate limiting - SLA based | 1.2.8
 | Spike Control | 1.1.5
 | Tokenization | 1.1.1
 | XML Threat Protection | 1.1.4
 |===
 
 For additional information about policies, refer to xref:api-manager::policies-landing-page.adoc[Policies].
 
-== Features Shipped with Anypoint Platform PCE Version 3.0
+== Features Shipped with Anypoint Platform PCE Version 3.1
 
 Refer to xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Release Notes] for detailed release information.
 
+Anypoint Platform PCE Version 3.1 contains the following features:
```

Comment:
```suggestion
Anypoint Platform PCE version 3.1 contains the following features:
```
I would capitalize this only in titles and headings. It's not really part of the name.